### ISSUE TITLE
Setting vibranceGUI to auto-start at boot causes explorer.exe to hang

### ISSUE THREAD

**Issue by kingrunes** [juvlarN/vibranceGUI#30]
After setting vibranceGUI to auto-start and rebooting, after 5-10 seconds from logging in explorer.exe freezes (spinning cursor on taskbar), the start menu won't open and some applications that have already started also freeze.

I can workaround this after each boot by launching Task Manager via Ctrl-Shift-Esc and restarting/killing explorer.exe. I have disabled auto-start, reboot and confirmed everything is fine multiple times, as soon as I enable auto-start the issue occurs.

A side-note which might be inaccurate/irrelevant is that after having this freeze occur, NVIDIA control panel (tray icon) isn't present, however after disabling auto-start its now present after booting. Given that vibranceGUI interfaces with the control panel I guess this could be related.

**Comment by juvlarN** [comment]
This is a problem caused by the notification popups of the .NET framework on Windows 10. 
Use the beta on my twitter, this is fixed in there.

**Comment by kingrunes** [comment]
Alright I'll have a look, thanks for the insanely quick reply :)

**Comment by kingrunes** [comment]
I've switched to the beta and now receive an error message when launching vibranceGUI: System.BadImageFormatException (see http://i.imgur.com/hbsr86G.png).

I've installed the Microsoft Visual C++ Redistributable 2012 package and removed old nvapi.dll files from C:\Windows after hearing that had fixed it for others (except the one that's in use by most running applications). I'm assuming there's a simple solution, but I haven't found it.

Cheers,
